main.py

```diff
@@ -1,19 +1,21 @@
 from workshops.numeral_systems import convert
 from workshops.conditions_and_loops import solve_the_knight_move_problem
 from workshops.real_numbers_and_strings import interest_rate_wrapper
+from workshops.zoom_link import zoom
 
 greeting = '''Choose option:
 1. Convert number from any base to any base.
 2. Solve the knight move problem
 3. Solve interest rate problem
-4. Exit
+4. Extract ID from Zoom link
+0. Exit
 '''
 
 
 def pass_input(func):
-    task_input = input("Provide input for the task: \n")
-    print("Answer:\n")
-    print(func(task_input))
+    task_input = input("\nProvide input for the task: \n")
```

Comment:
New line characters should be at the end of the previous output, not at the start of input. The idea is good though